spec/lib/zendesk_apps_tools/deploy_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+require 'zendesk_apps_tools/deploy'
+require 'json'
+
+describe ZendeskAppsTools::Deploy do
+  let(:mock_response) do
+    Class.new do
+      attr_reader :body
+      def initialize(json_body)
+        @body = json_body
+      end
+
+      def get(uri)
+        self
+      end
+    end
+  end
+
+  let(:subject_class) do
+    Class.new do
+      include ZendeskAppsTools::Deploy
+      attr_reader :get_connection
+
+      def initialize(get_response)
+        @get_connection = get_response
+      end
+
+      def connection(multipart = nil)
+        get_connection
+      end
+
+      def say_status(command, text)
+      end
+
+      def get_value_from_stdin(text)
+        'app_2'
+      end
+
+      def say(message, color = nil)
+      end
```

Comment:
Could we replace most of these with ` allow(subject).to receive_message`?

Reply:
Yes we can